style.css

```diff
@@ -0,0 +1,105 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&display=swap');
+
+body,
+html {
+  min-width: 100vw;
+  position: absolute;
+}
```

Comment:
- [ ] Currently, I am seeing the extra margin around the body. I think if you remove the default margin from the body and add your customized margin `margin: 0` in your body style it will replace the default style. Kindly fix it to match it with the Figma template 🙂 

Your design 👇🏽 
<img width="375" alt="Screenshot 2022-12-01 at 10 39 08 PM" src="https://user-images.githubusercontent.com/24832846/205115983-6de094c0-d0d9-4c01-bae1-199cbf1cd8f4.png">